README.md

```diff
@@ -188,10 +188,6 @@ Call this method (with the relevant arguments, for this method’s parameters) a
 
 ### Data analysis
```

Comment:
Is there another Data analysis coming to go into this README? If not, this will be an empty header, so might as well remove it then.

Reply:
removed for now